src/components/interaction-designer/toolbar/BlockErrorsExpandable.vue

```diff
@@ -43,10 +43,12 @@ import {ErrorObject} from 'ajv'
 import {Prop} from 'vue-property-decorator'
 import {IFlow} from '@floip/flow-runner'
 import {namespace} from 'vuex-class'
+import {get, union} from "lodash"
 
 const flowVuexNamespace = namespace('flow')
+const validationVuexNamespace = namespace('validation')
 
-const DEFAULT_LIST_SIZE = 2
+const DEFAULT_LIST_SIZE = 3
```

Comment:
@bulatgab , FYI here, I increased this as we have more validation errors to display

Reply:
Sounds good!